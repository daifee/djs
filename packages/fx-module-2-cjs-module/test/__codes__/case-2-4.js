
fxDefine('case-7', () => {
  const case1 = fxRequire('case-1');

  const case2 = fxRequire('case-2');


  case2.fly();

  return { case1, case2 };
});
