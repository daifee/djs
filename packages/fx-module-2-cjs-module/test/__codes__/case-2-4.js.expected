"use strict";

const case1 = require('case-1');
const case2 = require('case-2');
case2.fly();
module.exports = {
  case1,
  case2
}